### ISSUE TITLE
How to access scope/container from JobFilterAttribute (t: support)

### ISSUE THREAD

**Issue by nganbread** [HangfireIO/Hangfire#919]
I want to access some `InstancePerBackgroundJob` dependencies from my `JobFilterAttribute filter

Is there a convenient way to do this?

**Comment by nganbread** [comment]
Hmmm looks like its not possible because the job scope hasnt event been created at this point

**Comment by MrAdam** [comment]
Bumping this, as we are running into issues with this as well. We have some parameters for our job that we want to insert into some dependencies that are scoped in lifestyle to per-job, but the dependency being injected in the IServerFilter is outside the job scope..

**Comment by nganbread** [comment]
Yeah I was trying to do the same, but if you have a look through the sources it doesn't look like an quick/easy fix. It's been 6 months since I last looked, so it may have changed since then
